### PR TITLE
C3DC-408 - Arranged the properties in the download CSV file per requirement

### PR DESCRIPTION
**Pull request by Senait7-ux**
C3DC-397 is also resolved as part of this bug fix.

### CODE REVIEW

**Comment by Senait7-ux** [comment]
https://github.com/CBIIT/bento-c3dc-frontend/pull/45

File: src/bento/dashboardTabData.js [hunk 1]
```diff
@@ -1206,21 +1192,21 @@ export const tabContainers = [
...
-        dataField: 'diagnosis_comment',
-        header: 'Diagnosis Comment',
-        display: false,
+        dataField: 'age_at_diagnosis',
+        header: 'Age at Diagnosis (days)',
+        display: true,
         tooltipText: 'sort',
         role: cellTypes.DISPLAY,
       },
+      // Extra Hidden Columns
```
Comment (huuaho) [inline src/bento/dashboardTabData.js:1209]: is this an extraneous comment?

Comment (Senait7-ux) [reply src/bento/dashboardTabData.js:1209]: I think the comment gives a hint that a hidden column can be added. As there is another way to do the same thing in bento, it gives clarity and avoids ambiguity.

Comment (iksheth) [reply src/bento/dashboardTabData.js:1209]: @huuaho @Senait7-ux Yes, I added that comment but now as I think now if we are re-arranging this comments would not be helpful. We can remove it.

Comment (Senait7-ux) [reply src/bento/dashboardTabData.js:1209]: Okay, will remove it.

Comment (iksheth) [reply src/bento/dashboardTabData.js:1209]: Thanks @Senait7-ux and @huuaho

**Review by iksheth — APPROVED** [review]

**Pull request merged**